operator/tests/central/basic/70-injected-cabundle.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
@0x656b694d perhaps I didn't look carefully when this PR was in review, but I only happened to notice this file (and its `basic-sc` counterpart) now. As far as I can tell, these files are useless, since there is nothing to check their effect, if any. The next test step just deletes the CR.

What was your intention here?

Reply:
Hm, dunno actually. I thought they are like example files. I don't even see how other files from the directory are used.

Reply:
They are declarative tests run by the [`kuttl` tool](https://kuttl.dev/). In short, each set with the same numbered prefix is a test step. The ones with `NN-error*` and `NN-assert*` are assertions, and the rest instruct `kuttl` what to apply on the cluster before checking the assertions.

Reply:
https://github.com/stackrox/stackrox/pull/3518